core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.html

```diff
@@ -111,14 +111,9 @@
           <div class="oppia-outer-container">
             <div class="oppia-mascot-container">
               <div class="oppia-mascot-avatar-container">
-                <a [ngClass]="{'oppia-disabled-image-link': (OPPIA_AVATAR_LINK_URL === null)}"
-                   [href]="OPPIA_AVATAR_LINK_URL"
-                   rel="noopener"
-                   target="_blank">
-                  <img class="oppia-mascot-avatar"
-                       [src]="OPPIA_AVATAR_IMAGE_URL"
-                       alt="Oppia Mascot Avatar">
-                </a>
+                <img class="oppia-mascot-avatar"
+                     [src]="OPPIA_AVATAR_IMAGE_URL"
+                     alt="Oppia Mascot Avatar">
```

Comment:
```suggestion
                     alt="">
```

Reply:
As the avatar is only visual element without a meaning it should not have an alt tag.

Reply:
ditto elsewhere